src/components/VirtualizedGrid/VirtualizedGrid.tsx

```diff
@@ -299,8 +299,7 @@ const VirtualizedGrid: VibeComponent<VirtualizedGridProps> = forwardRef(
                 onItemsRendered={onItemsRenderedCB}
                 className={scrollableClassName}
               >
-                {/*@ts-ignore*/}
-                {cellRenderer}
+                {cellRenderer as unknown as VibeComponent<GridChildComponentProps>}
```

Comment:
Not a fun of this one, but haven't found better solution, so let's consider this as good enough